app/controllers/openid_application_controller.rb

```diff
@@ -20,51 +20,49 @@ class OpenidApplicationController < ApplicationController
 
   private
 
-  def permit_scopes(scopes, actions: [])
```

Comment:
This method and several others are moved to a `# Helper methods` section at the bottom of the controller to help with organization.